draft-ekwk-capport-rfc7710bis.xml

```diff
@@ -222,13 +223,15 @@
             <t>option-code: The Captive-Portal DHCPv6Option (103) (two
             octets)</t>
 
-            <t>option-len: The length, in octets of the URI.</t>
+            <t>option-len: The unsigned 16-bit length, in octets, of the URI.
```

Comment:
Did you want to be consistent with the above?  Either both "unsigned 16-bit/8-bit length" or "length (one/two octet[s])"